android-client-sdk/src/main/java/com/devcycle/sdk/android/api/DVCClient.kt

```diff
@@ -61,8 +64,20 @@ class DVCClient private constructor(
             try {
                 fetchConfig(user)
                 isInitialized.set(true)
+                eventSource = EventSource.Builder(Handler(fun (event: String?, messageEvent: MessageEvent?) {
+                    val data = JSONObject(messageEvent?.data)
+                    var type = ""
+                    var lastModified: Long = 0
+                    try {
+                        type = data.get("type") as String
+                        lastModified = data.get("lastModified") as Long
+                    } catch (e: JSONException) {}
+
+                    if (type == "refetchConfig" || type == "") { // Refetch the config if theres no type
+                        refetchConfig(true, lastModified)
+                    }
+                }), URI(config?.sse?.url)).build()
```

Comment:
the event source should only be created if this URL exists in the config

Reply:
fixed!